ui/apps/platform/src/Containers/Network/Graph/Graph.js

```diff
@@ -132,8 +132,8 @@ class Graph extends Component {
             simulatedBaselines,
         } = this.props;
 
-        // If we have more than 1100 nodes, display a message instead of the graph.
-        const nodeLimit = 1100;
+        // If we have more than 2000 nodes, display a message instead of the graph.
+        const nodeLimit = 2000;
```

Comment:
What's the significance of 2000? Can customers have more than 2000 in a namespace too?